version.md

```diff
@@ -9,7 +9,28 @@ If this is set to true, the version can only be retrieved if the user is an admi
 
 ## Create version
 
-TODO
+**POST /api/versioning/versions[?summary=<summary-text>]**
```

Comment:
I'd also like more information on this endpoint.  What is the response returned by this POST?  Are you given a reference to a newly created Item or WorkspaceItem which corresponds to the new Version?

We also likely should better understand what creating a new Version of an Entity does to Relationships (as @benbosman notes).  Specifically, are Relationships copied to the new Entity or not.

Reply:
The returned object is a version I will add this information explicitly in the contract. I'm also noted that the return code must be 201 to be aligned with our general framework.

@benbosman @tdonohue versioning and entities/relationship interaction is a very good point, thanks to raise it. The obvious choice would be to duplicate the relationship as any other information in the original item is actually duplicated in the new (version) item but this will have the side effect to open at least a couple of tasks not covered in the original estimation and our plan: 
- relationship shown in the entity details page should  filter out older versions (the publication should only list the current version of the author profile) or filter out the current one if not yet archived
- adding multiple relationship among one item and several different versions of the other item will make problematic the computation of the position. A solution could be to put the new version just after the previous item in the relationship. Of course some changes will be required also on the virtual metadata code to ignore older / current version(s) as appropriate (similar to the public page discussion)

I see two options here:
1. we will ignore the relationship completely, this would mean that the versioning feature will work in DSpace 7 as in previous version for normal items. Entity item will be not supported at this stage
2. we can copy the relationship over the new version and create subsequent ticket for the additional needed cleanup. Please note that also if we decide to don't copy the relationship automatically in the new version is still possible that the user will restore the relationships that were present in the previous version manually, maybe during the submission of the new version

or maybe there are other options or documentation about how the entities were planned to work with the versioning feature when it was proposed/introduced that I have overlooked?

Personally, I would suggest to exclude entities item from the versioning feature for this first implementation (7.1)